mandatory/1-fix-functions.js

```diff
@@ -21,7 +21,7 @@ function getMood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = num >= 10;
```

Comment:
Looks good but here's a question: Is `num >= 10` going to assert that the num is greater than 10?